kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -231,9 +231,13 @@ const Filters: React.FC<FiltersProps> = ({
         props.seekType = SeekType.TIMESTAMP;
       }
 
+      const isSeekTypeWithSeekTo =
+        props.seekType === SeekType.TIMESTAMP ||
+        props.seekType === SeekType.OFFSET;
+
       if (
         selectedPartitions.length !== partitions.length ||
-        currentSeekType === SeekType.TIMESTAMP
+        isSeekTypeWithSeekTo
```

Comment:
doesn't this give us the issue where we had all the partitions selected, now the first part of the `if` condition is `false` but the second one because we are using `OR` will be true hence we will have that `url` length issue again during `offset` and `timestamp`

Reply:
@Mgrdich 
hi, this is my suggestion https://github.com/provectus/kafka-ui/issues/3525 to solve the `url length isssue`. could you teams review it?

Reply:
Hey @Mgrdich
I've made a short review of the @p-eye solution for not combining offset with partition - it seems this one should work fine, but it probably needs some task on the backend to support this kind of payload and a bit of refactoring on the client side

Reply:
@Mgrdich the suggested change doesn't work well with pagination. The decision, for now, is to fix search modes considering that we might break the URL length fix.